core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -749,6 +832,16 @@ class HttpGoogleServicesDAO(
         }).map { op =>
           rawlsBillingProjectOperation.copy(done = toScalaBool(op.getDone), errorMessage = Option(op.getError).map(error => toErrorMessage(error.getMessage, error.getCode)))
         }
+
```

Comment:
should we get rid of the above cases?

Reply:
in the case where there are leftover project creations when we deploy this, i want the error to be caught by [these cases](https://github.com/broadinstitute/rawls/pull/1041/files#diff-5fc8bf1e934648fb8daf9dce39334029R112). so this code should probably return a successful future with an error message or something

Reply:
but as in the next comment, we should ensure that there are no project creations in flight when we deploy this new code. I think we have 2 options
1) [easiest] have our code support only the new way and destroy all evidence of the old way
2) [more robust] first release supports both simultaneously but all new project creations use the new way then a follow on release that removes the old way

I don't think there should be any middle ground that recognizes the old way and errors.

Reply:
done